torch/_dynamo/guards.py

```diff
@@ -1078,7 +1084,7 @@ def convert(size_or_stride):
                 device_index = t.device.index
                 requires_grad = t.requires_grad
                 sizes = dynamic_dims_sizes[i]
-                strides = dynamic_dims_strides[i]
+                strides = dynamic_dims_strides[i] if not t.is_nested else "None"
```

Comment:
Maybe worth a comment on what the dynamic shapes strides strategy is for nested?